src/client/user-main.cc

```diff
@@ -15,6 +15,8 @@ DEFINE_string(conf_file,
               TOSTRING(PROJECT_DIRECTORY) "/data/userconfig_512.json",
               "Config filename");
 
+DEFINE_int32(ue_id, -1, "optional UE ID to select a user radio");
```

Comment:
How is this used?

Reply:
We already have hooks for this, you need an offset and a count.  Would be best to fix those instead of adding new values.